### DISCUSSION TITLE
Need Help with XPath

### DISCUSSION THREAD

**Discussion by Randalix** [FreshRSS/FreshRSS#5508] [Q&A] [answered]
I'm having a hard time creating my first feed with XPath. This is the website I'm trying to scrape [sidefx tutorials]( https://www.sidefx.com/tutorials/).
I've checked DanQs blog post, but my example is much more complex, and I'm missing a way to quickly test it.
If I do: `document.evaluate('//div[@class="card-box"]',document).iterateNext()`I get the parent div, but then I would need the firstChild img alt text as a title.
I'm just posting this on hope to get some hints to the right direction. Any help is much appreciated.

**Comment by Frenzie** [comment, marked as answer]
Does that site load anything at all without JS?

**Reply by Randalix** [reply]
No, actually not. I guess it is not possible then?
Thanks for your time.

**Reply by Frenzie** [reply]
Definitely not using that method then, no.
